### PR TITLE
Flake8

### PR DESCRIPTION
**Pull request by stephendonner**

### CODE REVIEW

File: utils.py [hunk 1]
```diff
@@ -4,6 +4,8 @@
 
 # Write to the same line dynamically with this.
 # Call finish_same_line() when you're done with the line.
+
```
Comment (gmierz) [inline utils.py]: I think we'll have to put the comments within the function to prevent this.

File: adb_utils.py [hunk 2]
```diff
@@ -77,24 +77,24 @@ def get_battery_level():
...
+                status: 2\n
+                health: 2\n
+                present: true\n
+                level: 96\n
+                scale: 100\n
+                voltage: 4387\n
+                temperature: 300\n
+                technology: Li-ion\n
+
+        """
```
Comment (gmierz) [inline adb_utils.py]: I don't know what you think but this looks a little funny (with the """ being an extra tab).

Comment (stephendonner) [reply adb_utils.py]: This is suspect; yeah.  Something smells fishy - I might have not branched for this PR cleanly...

**Comment by stephendonner** [comment]
An evil ```git push -f origin master``` by yours truly has resulted in this calamity; I'm closing and we'll do this The Right Way (tm).

**Pull request closed without merging**